tests/forms/widgets.py

```diff
@@ -49,6 +49,14 @@ def test_init(self):
         for key, value in kwargs.items():
             self.assertEqual(value, getattr(widget, key))
 
+    def test_obsolete(self):
+        kwargs = dict(template='checkbox', multiple=True)
+        with self.assertRaises(TypeError) as exc:
+            widgets.Widget(**kwargs)
```

Comment:
What `kwargs` is constracted for? Why not just `widgets.Widget(template='checkbox', multiple=True)`?